### PR TITLE
Crew Credits Should Now Correctly Display Antagonists

### PR DESCRIPTION
**Pull request by Mister-Moriarty**
[UI] [Bug]

## About the PR:
The "Antagonists" section of the crew credits has been amended to display all antagonist roles that a mind possesses, and amended so to display an antagonist mind's associated job in other sections of the credits, as opposed to repeating their antagonist role.

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
Would make sense to display both their job title and antagonist role in the crew section, otherwise you have to cross reference to check if they were an antag or not.

File: _std/lists.dm [hunk 1]
```diff
@@ -244,7 +244,7 @@ proc/keep_truthy(some_list)
 		.[first[i]] = second[i]
 
 /// Returns a list in plain english as a string
-/proc/english_list(list/input, nothing_text = "nothing", and_text = " and ", comma_text = ", ", final_comma_text = "" )
+/proc/english_list(list/input, nothing_text = "nothing", and_text = " and ", comma_text = ", ", final_comma_text = "," )
```
Comment (ZeWaka) [inline _std/lists.dm:247]: 😻😻😻

**Review by ZeWaka — APPROVED** [review]
